src/modules/endpoint/endpoints/__tests__/categories.spec.js

```diff
@@ -48,34 +47,39 @@ describe('categories', () => {
     status: 'trash'
   }]
 
-  const categoryModels = [
-    new CategoryModel({
-      id: 3650,
-      url: '/augsburg/de/anlaufstellen',
-      title: 'Anlaufstellen zu sonstigen Themen',
-      content: '',
-      parentId: 0,
-      parentUrl: '/augsburg/de',
-      order: 75,
-      availableLanguages: {
-        en: 4361, ar: 4367, fa: 4368
-      },
-      thumbnail: 'https://cms.integreat-ap…/03/Hotline-150x150.png'
-    }),
-    new CategoryModel({
-      id: 3649,
-      url: '/augsburg/de/willkommen',
-      title: 'Willkommen',
-      content: '',
-      parentId: 0,
-      parentUrl: '/augsburg/de',
-      order: 11,
-      availableLanguages: {
-        en: 4804, ar: 4819, fa: 4827
-      },
-      thumbnail: 'https://cms.integreat-ap…03/Beratung-150x150.png'
-    }),
-    new CategoryModel({id: 0, url: '/augsburg/de', title: 'augsburg'})
+  const categoryModels = [{
```

Comment:
Types get lost here. Would be nice to have them when initially creating this array.

Reply:
is it necessary to have types in tests?

Reply:
Tests have to be stable too so I'd say yes. @Schedulaar @NKlug what do you think?

Reply:
Hmm... I'm not sure... In this case: If categoryModels was 'misstyped', flow would (should) complain in line 95 (https://github.com/Integreat/integreat-webapp/pull/112/files/231674fb44fb954fe0c7d41676b50ad5b757ce72#diff-d86c55c5be132abd301ff00fb6eee652R95), that categoryModels can't be inserted into `new CategoriesMapModel()`. So I guess we're good to go

Reply:
I could create a task to add flow to all tests @maxammann ?